packages/fast-components-styles-msft/src/checkbox/index.ts

```diff
@@ -99,12 +97,11 @@ const styles: ComponentStyles<ICheckboxClassNameContract, IDesignSystem> = {
             return config.foregroundColor;
         },
         ...applyTypeRampConfig("t7"),
-        marginLeft: toPx(5),
-        marginTop: toPx(2)
+        marginLeft: toPx(5)
     },
     checkbox_disabled: {
+        cursor: "not-allowed",
         "& $checkbox_input, & $checkbox_label": {
```

Comment:
Add `, & $checkbox_span` to the `"& $checkbox_input, & $checkbox_label"` selector

Reply:
I moved the opacity to the top level to reduce the selector chain.